js/README.md

```diff
@@ -42,7 +42,7 @@ See [Tweet Entities](https://dev.twitter.com/overview/api/entities-in-twitter-ob
 
 ## Remaining character count example
 
-    var tweet = "This is a test tweet":
+    var tweet = "This is a test tweet";
```

Comment:
@outoftime 

- [Google style guide](https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml): "Always use semicolons"
- [Stack Overflow](http://stackoverflow.com/questions/444080/do-you-recommend-using-semicolons-after-every-statement-in-javascript): "Yes, you should use semicolons after every statement in JavaScript."
- [Airbnb style guide](https://github.com/airbnb/javascript#semicolons): "Yup"